<launch>
  <!--
    load up the camera. To select a different camera then use `rosparam set
    cv_camera/device_id 1`
  -->
  <group ns="sensors">
    <include file="$(find tfr_sensor)/launch/realsense.launch"/>
  </group>

  <!-- load up the server -->
  <node type="aruco_action_server"  name="aruco_action_server" pkg="tfr_aruco"
    output="screen"/>

  <!-- load up the client and remap so that it gets the right images -->
  <remap from="/image_raw" to="/camera/color/image_raw" />
  <remap from="/camera_info" to="/camera/color/camera_info" />
  <node type="tfr_aruco_example_client" name="tfr_aruco_example_client"
    pkg="tfr_aruco" output="screen" />
</launch>
